app/models/suggestion.rb

```diff
@@ -0,0 +1,7 @@
+class Suggestion < ApplicationRecord
+  # Remember to create a migration!
+  belongs_to :user
+  validates :title, presence: true
+  validates :description, presence: true
+  validates :user_id, presence: true
```

Comment:
Even better:
```ruby
validates :user, presence: true
```
Question: why is this better?

Reply:
Corrected now.
Because it checks if the join with an existing user in table User can be made?